geomesa-blobstore/geomesa-blobstore-core/src/test/resources/log4j.xml

```diff
@@ -1,12 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-  ~ Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
-  ~ All rights reserved. This program and the accompanying materials
-  ~ are made available under the terms of the Apache License, Version 2.0 which
-  ~ accompanies this distribution and is available at
-  ~ http://www.opensource.org/licenses/apache2.0.php.
-  -->
-
```

Comment:
we don't have copyrights in our other logging files - we discussed at one point and decided no need to copyright things that don't show any 'originality'